lib/fuse_i.h

```diff
@@ -43,28 +43,28 @@ struct fuse_notify_req {
 
 struct fuse_session {
 	char *mountpoint;
-	volatile int exited;
-	int fd;
+	volatile int32_t exited;
+	int32_t fd;
 	struct fuse_custom_io *io;
 	struct mount_opts *mo;
-	int debug;
-	int deny_others;
+	int32_t debug;
```

Comment:
if we're forcing recompilation anyways, does it make sense to rework some of these fields like for example having debug, got_init, got_destroy, deny_others, etc. to be a bool/uint8_t or a bit instead of a int32_t?

Reply:
Thank you! fuse_i.h was actually accidental and not intended. We should be able to update it anytime, but I took your suggestion and reworked it a bit. Also made fields cache line aligned now.

Reply:
Should we also roll in your "struct timespec" changes from https://github.com/libfuse/libfuse/pull/1032 while we're breaking ABI?